packages/core/src/__mocks__/languages.ts

```diff
@@ -402,3 +404,88 @@ export const LANGUAGES: Record<
         },
     },
 };
+
+export const LANGUAGES_2: MockLanguages = {
```

Comment:
It feels weird having those two objects here. Could we update the tests to work with just a single object?
We can always assemble the languages array manually for each test (not just passing the whole `LANGUAGES`), which would give more control